trunner/target/armv7a9.py

```diff
@@ -89,7 +89,7 @@ def from_context(cls, ctx: TestContext):
     def flash_dut(self):
         plo_loader = ZynqGdbPloLoader(
             dut=self.dut,
-            script=f"{self._project_dir()}/phoenix-rtos-build/scripts/upload-zynq7000.gdb",
+            script=f"{self._project_dir()}/phoenix-rtos-build/scripts/upload-zynq7000-smt2.gdb",
```

Comment:
I feel that with this change `class ARMv7A9Target` loses it's previous meaning, as now it only gives support for ARMv7A9 targets that have smt2 controller on board, eg I doubt this class could be reused to test zturn target. Said that I think that class structure needs rethinking

Reply:
You are right, it can may be misleading. That's why I'd suggest leaving there:

```
    @abstract_method
    def flash_dut(self):
        # for now there is no common flashing approach for all future armv7a9 targets

```

and move the valid flash_dut() to Zynq7000ZedboardTarget.

@badochov what do you think?
@maska989 could you check this solution?

Reply:
Seems like a good solution to me

Reply:
I'll approve PR once this change is made

Reply:
@badochov done